src/js/tabs/setup.js

```diff
@@ -416,35 +415,76 @@ setup.initialize = function (callback) {
             }
         };
 
+        function showDialogBuildInfo(title, message) {
+            const dialog = $('.dialogBuildInfo')[0];
+
+            $('.dialogBuildInfo-title').html(title);
+            $('.dialogBuildInfo-content').html(message);
+
+            if ( ! dialog.hasAttribute('open')) {
+                dialog.showModal();
+                $('.dialogBuildInfo-closebtn').on('click', function() {
+                    dialog.close();
+                });
+            }
+        }
+
+        function buildGrid(list, columns, classContainer, classItem) {
+            let rows = Math.ceil(list.length / columns);
+            let grid = `<div class='${classContainer}'><table>`;
+
+            for (let i = 0; i < rows; i++) {
+                grid += "<tr>";
+                for (let j = 0; j < columns; j++) {
+                    let index = i * columns + j;
+                    if (index < list.length) {
+                        grid += `<td><div class='${classItem}'>${list[index]}</div></td>`;
+                    } else {
+                        grid += "<td></td>"; // empty cell, to ajust
+                    }
+                }
+                grid += "</tr>";
+            }
+            grid += "</table></div>";
+
+            return grid;
+        }
```

Comment:
I prefer implementing my earlier suggestion instead. As the code is much leaner and as this is using tables while we are using grid layout does not make sense. The code provided is dynamic on itself and uses css for presentation.

Reply:
See https://github.com/betaflight/betaflight-configurator/pull/3770#issuecomment-1938932576

Reply:
Now get your idea, but without table I have only an one collum layout, missing empty cell `<td></td>` 
`<div class='${classItem}>&nbsp;</div>` or `<div>&nbsp;</div>` won't give the same
Do you know how to have an empty placeholder with `div` element

Reply:
Now you get what you want :-)
![image](https://github.com/betaflight/betaflight-configurator/assets/99370924/8f9b358b-e9f6-4836-929a-fa643fced297)